mode/mode_remote.go

```diff
@@ -23,7 +23,7 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
-const installerProgramTemplate = `#!/usr/bin/env sh
+const installerProgramTemplate = `#!/usr/bin/env bash
```

Comment:
I can see where the problem is here. The script claims to use `sh` while using `[[ ... ]]`, which is `bash`. The preferred solution would be to fix the script so that it does not require `bash`.